### PR TITLE
get yelp shops:                                   

### LINKED ISSUE #7: Get current Latitude and Longitude
Use this reference to collect the users lat and long: https://stackoverflow.com/questions/24906833/how-to-access-current-location-of-any-user-using-python

### LINKED ISSUE #13: Get shops from Yelp

### LINKED ISSUE #14: Build Yelp API Test

### PR DESCRIPTION
**Pull request by amylynndee**
corrected environment variable from previous branch, initial structure of the api call

### CODE REVIEW

**Comment by amylynndee** [comment]
@Mr-3rd  and @pengxuemoua please review my changes and let me know if there is anything I can fix, thank you!

File: apis/shops_api.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import requests
+import geocoder
+import logging
+import os
+
+def get_location():
+    g = geocoder.ip('me')
+    return g.latlng
+
+get_location()
```
Comment (Mr-3rd) [inline apis/shops_api.py]: @amylynndee, we can remove this empty reference to the locations

**Review by Mr-3rd — CHANGES_REQUESTED** [review]
@amylynndee We can remove the location call on line 17

File: apis/shops_api.py (hunk 1 above)
Comment (Mr-3rd) [reply apis/shops_api.py]: Thanks for removing~!

**Review by Mr-3rd — APPROVED** [review]

**Review by pengxuemoua — APPROVED** [review]
API looks good. Approved to merge into main.

**Pull request merged**